packages/graphqlgen/src/generators/ts-generator.ts

```diff
@@ -303,8 +303,10 @@ function renderResolverTypeInterface(
   modelMap: ModelMap,
   context?: ContextDefinition,
 ): string {
+  // TODO add types for key and value
   return `
   export interface Type {
+    [key: string]: any;
```

Comment:
I would really like to avoid adding this as this massively weakens the type-safety of the generated resolvers. I believe that we'll need to introduce a change to the types directly in `graphql-tools`.